wordcloud/wordcloud.py

```diff
@@ -507,9 +508,20 @@ def process_text(self, text):
 
         flags = (re.UNICODE if sys.version < '3' and type(text) is unicode
                  else 0)
-        regexp = self.regexp if self.regexp is not None else r"\w[\w']+"
 
+        # the regex used to detect words is a combination of normal words, ascii art, and emojis
```

Comment:
I would rather not include such a complex regexp. You can just pass it as parameter for the example, right?